File: scoutingapp/src/main/java/org/usfirst/frc/team25/scouting/data/models/TeleOp.java
package org.usfirst.frc.team25.scouting.data.models;


/**
 * Container holding data from the tele-operated period
 * Includes endgame data
 */
public class TeleOp {

    private int netTeleop;
    private int minFoulTeleop;
    private int majFoulTeleop;
    private int levelOneTeleop;
    private int levelTwoTeleop;
    private int levelThreeTeleop;
    private int levelFourTeleop;
    private int coralCount;
    private int processorTeleop;
    private int missedTeleop;

    public TeleOp(int netTeleop, int minFoulTeleop, int majFoulTeleop, int levelOneTeleop, int levelTwoTeleop, int levelThreeTeleop, int levelFourTeleop, int coralCount, int processorTeleop, int missedTeleop) {
        this.netTeleop = netTeleop;
        this.minFoulTeleop = minFoulTeleop;
        this.majFoulTeleop = majFoulTeleop;
        this.levelOneTeleop = levelOneTeleop;
        this.levelTwoTeleop = levelTwoTeleop;
        this.levelThreeTeleop = levelThreeTeleop;
        this.levelFourTeleop = levelFourTeleop;
        this.coralCount = coralCount;
        this.processorTeleop = processorTeleop;
        this.missedTeleop = missedTeleop;
    }

    public int getNetTeleop() {return netTeleop;}

    public int getMinFoulTeleop() {return minFoulTeleop;}

    public int getMajFoulTeleop() {return majFoulTeleop;}

    public int getLevelOneTeleop() {return levelOneTeleop;}

    public int getLevelTwoTeleop() {return levelTwoTeleop;}

    public int getLevelThreeTeleop() {return levelThreeTeleop;}

    public int getLevelFourTeleop() {return levelFourTeleop;}

    public int getCoralCount() {return coralCount;}

    public int getProcessorTeleop() {return processorTeleop;}

    public int getMissedTeleop() {return missedTeleop;}
}

File: scoutingapp/src/main/java/org/usfirst/frc/team25/scouting/data/models/TeleOp.kt
package org.usfirst.frc.team25.scouting.data.models

/**
 * Container holding data from the tele-operated period
 * Includes endgame data
 */
data class TeleOp(val cargoShipHatches: Int,
                  val rocketLevelOneHatches: Int,
                  val rocketLevelTwoHatches: Int,
                  val rocketLevelThreeHatches: Int, val cargoShipCargo: Int, val rocketLevelOneCargo: Int,
                  val rocketLevelTwoCargo: Int, val rocketLevelThreeCargo: Int, val hatchesDropped: Int,
                  val cargoDropped: Int, val isAttemptHabClimb: Boolean, val attemptHabClimbLevel: Int,
                  val isSuccessHabClimb: Boolean, val successHabClimbLevel: Int,
                  val isClimbAssistedByPartner: Boolean, val assistingClimbTeamNum: Int,
                  val numPartnerClimbAssists: Int, val partnerClimbAssistEndLevel: Int,
                  val partnerClimbAssistStartLevel: Int)